packages/plugins/content/slate/src/plugins/paragraph/node.js

```diff
@@ -21,36 +21,13 @@
  */
 
 import React, { Component } from 'react'
-import { Placeholder } from 'slate'
-import { placeholder } from '../../const.js'
-import shallowEqual from 'fbjs/lib/shallowEqual'
 
 class Paragraph extends Component {
-  shouldComponentUpdate(nextProps) {
```

Comment:
A particular reason for removing `shouldComponentUpdate`?

Reply:
The way react decides if to update component is actually shallowEqual so it seemed redundant to do it twice. Unless there's something I am missing.